include/wx/translation.h

```diff
@@ -57,6 +59,12 @@
 #define wxGETTEXT_IN_CONTEXT_PLURAL(c, sing, plur, n) \
     wxGetTranslation((sing), (plur), n, wxString(), c)
 
+#define wxGETTEXT_PLATFORM_CONTEXT(c) \
+    (wxPlatformId::GetCurrent() + '.' + c)
```

Comment:
`wxS()` is not needed (and, in fact, wouldn't work) around _character_ (as opposed to string) constants.

Reply:
Perhaps not here then (a line which may get deleted anyway...) with some string config I haven't tried. For me with `wxS()` it did at least compile before. I checked with MSW now: `wxString`'s `wchar_t, size_t nRepeat` ctor is used, otherwise `char, size_t`. Also there's  quite some `wxS('` hits, all in library code, and I've been using it myself for some time.